docs/how-to-use.md

```diff
@@ -218,19 +218,19 @@ metadata:
   name: cvmfs-atlas-nightlies
 spec:
   containers:
-   - name: idle
-     image: busybox
-     imagePullPolicy: IfNotPresent
-     command: [ "/bin/sh", "-c", "trap : TERM INT; (while true; do sleep 1000; done) & wait" ]
-     volumeMounts:
-       - name: my-cvmfs-atlas-nightlies
-         mountPath: /atlas-nightlies
-         # Note that unlike in pod-single-repo-subpath.yaml, in this
-         # case we don't set mountPropagation nor subPath.
+    - name: demo
+      image: busybox
+      imagePullPolicy: IfNotPresent
+      command: [ "/bin/sh", "-c", "trap : TERM INT; (while true; do sleep 1000; done) & wait" ]
+      volumeMounts:
+        - name: my-cvmfs
+          mountPath: /my-cvmfs
+          # CVMFS automount volumes must be mounted with HostToContainer mount propagation.
```

Comment:
If this demo doesn't work as the comment says, then that's a bug in the driver, not an issue with docs. You can keep the indentation fix though!

Reply:
I.e. it *should* be possible to mount without `HostToContainer` if the PVC is mounted with `repository` parameter.

Reply:
Sorry, I copy-pasted the wrong snippet!

Reply:
Updated. Could you take another look?